### PR TITLE
fixed a test

### PR DESCRIPTION
**Pull request by nayimovag**
fixed [C350727](https://foliotest.testrail.io//index.php?/cases/view/350727)

### CODE REVIEW

File: cypress/e2e/export-manager/search-filter-verify.cy.js [hunk 1]
```diff
@@ -63,9 +68,19 @@ describe('export manager', () => {
...
+        cy.visit(TopMenu.bulkEditPath);
       });
 
+    // Creating a bulk edit job
+    FileManager.createFile(`cypress/fixtures/${userUUIDsFileName}`, userData.userId);
+    BulkEditSearchPane.checkUsersRadio();
+    BulkEditSearchPane.selectRecordIdentifier('User UUIDs');
+    BulkEditSearchPane.uploadFile(userUUIDsFileName);
+    BulkEditSearchPane.waitFileUploading();
+    BulkEditActions.downloadMatchedResults(matchRecordsFileName);
```
Comment (MaksimTerekhov) [inline cypress/e2e/export-manager/search-filter-verify.cy.js:80]: Seems that is excess. I assume you can remove this line

File: cypress/e2e/export-manager/search-filter-verify.cy.js [hunk 2]
```diff
@@ -16,13 +16,17 @@ import SearchPane from '../../support/fragments/circulation-log/searchPane';
...
+import BulkEditActions from '../../support/fragments/bulk-edit/bulk-edit-actions';
 
 let userData = {};
 const testData = {};
 const itemData = {
   barcode: getRandomPostfix(),
   instanceTitle: `Instance ${getRandomPostfix()}`,
 };
+const userUUIDsFileName = `userUUIDs_${getRandomPostfix()}.csv`;
+const matchRecordsFileName = `matchedRecords_${getRandomPostfix()}.csv`;
```
Comment (MaksimTerekhov) [inline cypress/e2e/export-manager/search-filter-verify.cy.js:29]: You can remove this line

File: cypress/e2e/export-manager/search-filter-verify.cy.js (hunk 1 above)
Comment (nayimovag) [reply cypress/e2e/export-manager/search-filter-verify.cy.js]: you're right, fixed

File: cypress/e2e/export-manager/search-filter-verify.cy.js (hunk 2 above)
Comment (nayimovag) [reply cypress/e2e/export-manager/search-filter-verify.cy.js]: done

**Review by MaksimTerekhov — APPROVED** [review]

**Review by re-jas — APPROVED** [review]

**Review by IhorBohdan — APPROVED** [review]

**Pull request merged**